### PR TITLE
Add fragments to repo activity

### PR DESCRIPTION
**Pull request by s0ys4uc3**
codeFragment and commitFragment

### CODE REVIEW

File: app/src/main/java/vn/edu/usth/minigh/closedtabFragment.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+public class closedtabFragment extends Fragment {
+
+    public closedtabFragment() {
```
Comment (McSinyx) [inline app/src/main/java/vn/edu/usth/minigh/closedtabFragment.java:13]:
```suggestion
public class ClosedTabFragment extends Fragment {

    public ClosedTabFragment() {
```

Also I don't think we need any of the methods below.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,3 +1,8 @@
 <resources>
-    <string name="app_name">miniGH</string>
+    <string name="app_name">MiniGH</string>
```
Comment (McSinyx) [inline app/src/main/res/values/strings.xml:2]:
```suggestion
    <string name="app_name">miniGH</string>
```

**Review by McSinyx — COMMENTED** [review]
Seems like a lot of cool stuff I could use here!  Could you please name classes in PascalCase?

File: app/src/main/java/vn/edu/usth/minigh/closedtabFragment.java (hunk 1 above)
Comment (s0ys4uc3) [reply app/src/main/java/vn/edu/usth/minigh/closedtabFragment.java:13]: ok I'll change all the classes name in PascalCase then push.

**Review by PhuongPhg — COMMENTED** [review]
U can use dimen for textSize to change it easier, otherwise it's look good to me

**Comment by McSinyx** [comment]
Would it be convenient for you to use font awesome, @s0ys4uc3?

**Comment by s0ys4uc3** [comment]
It's ok, I'll push it tonight w/ 2 last frags

**Comment by s0ys4uc3** [comment]
@PhuongPhg I already pushed 30m ago, waiting for reviews.

**Comment by 9a24f0** [comment]
how come it's 1k3 LoC o.O

**Comment by 9a24f0** [comment]
OYYYY I found the way to make this work

**Review by 9a24f0 — APPROVED** [review]
Thank god I have review permission here )-:

**Pull request merged**